src/styles/colors.ts

```diff
@@ -1,12 +1,13 @@
 const COLORS = {
   attorneyColor: '#85b4d2',
-  abaBlue: '#0069A9',
+  primaryBlue: '#0069A9',
   dark: '#292929',
-  interpreterColor: '#8dd89f',
+  tagColor: '#CCE1EE',
```

Comment:
I think on Figma, David calls it `light primary`, do we want to stick with that or call it `primaryLight`?

Reply:
is this not called `blueLight` in #30

Reply:
yes this is outdated comment, i shall resolve